nephio-ansible-install/NIND.md

```diff
@@ -0,0 +1,55 @@
+# nephio-in-docker (nind)
+
+## tldr
+
+nephio-in-docker (nind) packages the nephio-ansible-install into a [Docker](https://docs.docker.com/get-docker/) container following the setup used in the [nephio ONE summit 2022 workshop](https://github.com/nephio-project/one-summit-22-workshop).
+In addition nind also runs a [VS Code OSS](https://github.com/microsoft/vscode) workstation instance providing a browser based development and test enviroment.
+
+To build and run nephio-in-docker execute the commands listed below:
+
+```bash
+git clone https://github.com/nephio-project/one-summit-22-workshop.git
+cd one-summit-22-workshop/nephio-ansible-install
+docker build -t nind .
```

Comment:
Would be great if we can provide instructions for podman as well. 
If it cannot be done in this PR, it might be good to open an issue (somebody else can open a PR and provide podman instructions) to track this.
Thoughts?

Reply:
Do we know if there is a pod man in padman option, as this uses docker in docker?

Reply:
+1
however, don't know about podman to comment with insights and agree it probably is best to cover that as part of a different PR (solicited via an Github issue as you @vjayaramrh suggested)

Reply:
Sounds good. Would you reckon this could fall also under your [issue 108](https://github.com/nephio-project/one-summit-22-workshop/issues/108) (_"add support in ansible playbooks to install podman"_) that you already raised (ie not just installing but also showing how it is run)?

Reply:
The issue was opened last month to track only install of podman on VM. 
For NIND, I am guessing, the instructions for building the image, running the container are on a host (VM, BareMetal, workstation, etc) with docker already installed. If the host has podman installed of docker, we would want similar podman instructions. I believe they are separate issues in my opinion.

Reply:
Thanks @vjayaramrh , I added [issue 122](https://github.com/nephio-project/one-summit-22-workshop/issues/122) as a placeholder for the idea of implementing a similar concept with podman.